test/integration/basic-server.spec.js

```diff
@@ -27,56 +27,56 @@ const Static = Dispatcher.Static
 let server = null
 
 
-  let getData = function(data,timeout) {
-    return new Promise(function(resolve, reject) {
-      if (!timeout) {
+let getData = function(data, timeout) {
+  return new Promise(function(resolve, reject) {
+    if (!timeout) {
+      resolve(data);
+    } else {
+      setTimeout(function() {
         resolve(data);
-      }else{
-        setTimeout(function(){
-          resolve(data);
-        },timeout);
-      }
-    });
-  }
+      }, timeout);
+    }
+  });
+}
 
-  describe("Basic Http Server", function() {
+describe("Basic Http Server", function() {
 
-    before(function(done){
+  before(function(done) {
 
-      this.timeout(5000)
+    this.timeout(5000)
 
-      Helpers.load(path.join(__dirname,'./package.json'))
-      let env = new Env(Helpers)
-      let namespace = new Namespace(env,Helpers)
+    Helpers.load(path.join(__dirname, './package.json'))
+    let env = new Env(Helpers)
+    let namespace = new Namespace(env, Helpers)
 
-      let view = new View(Helpers,env)
-      let response = new Response(view)
-      let session = new Session(Helpers,{
-        get: function() {
-          return 'cookie'
-        }
-      })
-      server = new Dispatcher.Server(Routes,Request,response,Logger,session)
-      namespace.autoload()
+    let view = new View(Helpers, env)
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Too many errors. (11% scanned).